launch/Mako_G-319.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+  <arg name="name" default="camera" doc="The name of the camera"/>
+  <arg name="frame_id" default="camera" doc="The frame id of the camera"/>
+  <arg name="ip" default="" doc="The IP for the camera to connect to"/>
+  <arg name="guid" default="" doc="The GUID for the camera to connect to"/>
+  <arg name="camera_info_url" default="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343290.yaml"/>
+  <arg name="acquisition_rate" default="10" doc="Camera framerate (Hz)"/>
+
+  <include file="$(find avt_vimba_camera)/launch/mono_camera.launch">
+    <arg name="name" value="$(arg name)"/>
+    <arg name="frame_id" value="$(arg frame_id)"/>
+    <arg name="ip" value="$(arg ip)"/>
+    <arg name="guid" value="$(arg guid)"/>
+    <arg name="camera_info_url" value="$(arg camera_info_url)"/>
+    <arg name="acquisition_rate" value="$(arg acquisition_rate)"/>
+
+    <!-- Params specific to Mako G-319 -->
+    <arg name="pixel_format" value="BayerRG8"/>
+    <arg name="width" value="2064"/>
+    <arg name="height" value="1544"/>
+    <arg name="stream_bytes_per_second" value="100000000"/>
```

Comment:
Was this value actually calculated or is it an estimate?
(Bandwidth = Width x Height x Bytes Per Pixel x Frame Rate x Number of Cameras)

Reply:
I was just curious where you got the number! I'm OK either way